app/util/DragHandler.ts

```diff
@@ -207,11 +207,7 @@ export class DragHandler {
       return false;
     }
 
-    if (b.offset + b.length < a.offset) {
-      return false;
-    }
-
-    return true;
+    return b.offset + b.length >= a.offset;
```

Comment:
Same thing as above, checks returning `false`, with a return `true` at the end.  Breaks the pattern, this isn't more readable.